### DISCUSSION TITLE
Websocket support

### DISCUSSION THREAD

**Discussion by seandlg** [upptime/upptime#290] [Ideas]
I apologize in advance if I have missed this feature: Is there any way Upptime supports checking websocket connections?

I've seen the `tcp-ping` feature, which is great already! I'll stick with this for now, though it'd be great to be able to send a `greet` via Websocket and check the reponse :)

**Comment by ooliver1** [comment]
hi @seandlg there seems to be a `ws` check and a `body` key you can use, i was unaware of this too and im surprised its not in the docs

edit: body key seems to never work

**Reply by seandlg** [reply]
Ohh splendid! Thanks for sharing the information :)
